dev/com.ibm.ws.jaxws.2.3.webcontainer/src/com/ibm/ws/jaxws23/webcontainer/JaxWsHttpServletRequestAdapter.java

```diff
@@ -41,7 +42,7 @@
  * as in EJB based Web Services, the EJB invocation context is built very earlier in one intercepter, which may cause issue for those methods requiring
  * Web Component Context MetaData
  */
-public class JaxWsHttpServletRequestAdapter implements HttpServletRequest {
+public class JaxWsHttpServletRequestAdapter extends HttpServletRequestWrapper implements HttpServletRequest {
```

Comment:
So you didn't carry this over to the 2.2 version.  So it isn't needed?

Reply:
This was an attempt to mimic what the jaxrs-2.1 is doing with more updated version of the spec. Since this change wasn't needed in the first place, I've removed it along with all the other duplicate code in the ...jaxws.2.3.webcontainer.